consensus/types/src/chain_spec.rs

```diff
@@ -2620,6 +2632,19 @@ mod yaml_tests {
             default_max_blobs_per_block_electra()
         );
         assert_eq!(spec.max_blobs_per_block_within_fork(ForkName::Fulu), 20);
+
+        // Check that serialization is in ascending order
+        let yaml = serde_yaml::to_string(&spec.blob_schedule).expect("should serialize");
+
+        // Deserialize back to Vec<BlobParameters> to check order
```

Comment:
It's a little weird that a serde round trip result in different ordering. Maybe we should reverse again in deserialize?